page-index/page-index.js

```diff
@@ -4,11 +4,14 @@ const timescaleCloudPageIndex = require("../cloud/page-index/page-index");
 const timescaleMSTPageIndex = require("../mst/page-index/page-index");
 const promscalePageIndex = require("../promscale/page-index/page-index");
 const installPageIndex = require("../install/page-index/page-index");
-const functionsPageIndex = require("../functions/page-index/page-index");
+const pgcommandsPageIndex = require("../pg-commands/page-index/page-index");
```

Comment:
@jacobprall can correct me if I'm wrong on this, but I think this rename will require changes to the web-documentation code that made the top-level section

Reply:
I believe this has been done. @jacobprall can correct me if I'm wrong, though.